src/components/tabsConfig.jsx

```diff
@@ -0,0 +1,56 @@
+// src/tabsConfig.js
+
+import PIESCheckin from './tabs/PIESCheckin';
+import Growth from './tabs/Growth';
+import Tips from './tabs/Tips';
+import Events from './tabs/Events';
+import Memberships from './tabs/Memberships';
+import Favorites from './tabs/Favorites';
+import Production from './tabs/Production';
+
+export const tabMap = {
+    checkin: {
+        label: 'PIES Checkin',
+        component: <PIESCheckin />,
```

Comment:
I encountered some rendering errors when trying to pass dynamic tabs that were components instead of strings.

This library now allows for strings to easily reference their functional component.